tests/fixture_plugins/demultiplex_fixtures/flow_cell_fixtures.py

```diff
@@ -0,0 +1,172 @@
+"""Fixtures for flow cell objects."""
+from pathlib import Path
+
+import pytest
+
+from cg.constants.demultiplexing import BclConverter, DemultiplexingDirsAndFiles
+from cg.models.flow_cell.flow_cell import FlowCellDirectoryData
+
+# Functional flow cells
+
+
+@pytest.fixture(scope="module")
+def hiseq_x_single_index_flow_cell(
+    hiseq_x_single_index_flow_cell_dir: Path,
```

Comment:
Out of curiosity how is `hiseq_x_single_index_flow_cell_dir` found? Is it because it's in the same directory as this file?

Reply:
That fixture is defined in `tests/fixture_plugins/demultiplex_fixtures/path_fixtures.py`, which is called as a plugin in the root `conftest.py`. It is equivalent to defining the fixture in the contest, so it is available to all tests.